data/net/minecraft/world/level/levelgen/NoiseRouterData.mapping

```diff
@@ -1,16 +1,30 @@
 CLASS net/minecraft/world/level/levelgen/NoiseRouterData
 	METHOD bootstrap (Lnet/minecraft/data/worldgen/BootstapContext;)Lnet/minecraft/core/Holder;
 		ARG 0 context
+	METHOD caves (Lnet/minecraft/core/HolderGetter;Lnet/minecraft/core/HolderGetter;)Lnet/minecraft/world/level/levelgen/NoiseRouter;
+		ARG 1 noiseParameters
```

Comment:
The existing mappings for the methods in this class use `noiseParametersGetter`. Either this or those should be updated, for consistency.